packages/frontend/src/ee/features/aiCopilot/components/ChatElements/ArtifactButton/AiArtifactButton.tsx

```diff
@@ -23,46 +23,36 @@ export const AiArtifactButton: FC<AiArtifactButtonProps> = ({
     return (
         <UnstyledButton
             className={styles.artifactButton}
-            artifact-open={isArtifactOpen ? 'true' : undefined}
+            data-artifact-open={isArtifactOpen}
+            data-loading={isLoading}
             onClick={onClick}
             disabled={isLoading}
         >
-            <Group className={styles.container}>
-                <Group className={styles.content}>
+            <Box className={styles.container}>
+                {isLoading ? (
+                    <Loader size={14} color="gray.5" />
+                ) : (
+                    <MantineIcon
+                        icon={IconArtboard}
+                        size={14}
+                        className={styles.icon}
+                    />
+                )}
+
+                <Box className={styles.content}>
                     {isLoading ? (
-                        <Loader
-                            size={14}
-                            color="gray"
-                            className={styles.icon}
-                        />
+                        <Text size="sm" c="dimmed">
+                            Creating...
+                        </Text>
                     ) : (
-                        <MantineIcon
-                            icon={IconChartDots3}
-                            size={14}
-                            color="gray"
-                            className={styles.icon}
-                        />
+                        displayTitle && (
+                            <Text size="sm" fw={500} className={styles.title}>
+                                {displayTitle}
+                            </Text>
```

Comment:
we can also display description I think

Reply:
Ah yes! will do this on a separate PR as the layout needs to change a bit